proj/qtcreator/libxively-v3.includes

```diff
@@ -1,464 +1,264 @@
+../../src/import/tls/wolfssl/support
+../../src/import/tls/wolfssl/wolfcrypt/benchmark
+../../src/import/protobuf-c/test
 ../../src/import/tls/wolfssl/wrapper/CSharp/wolfSSL-TLS-PSK-Server/Properties
+../../src/libxively/io/fs/posix
+../../src/import/tls/wolfssl/mqx/wolfcrypt_benchmark
+../../src/import/tls/wolfssl/IDE/MDK5-ARM/Projects/CryptBenchmark/RTE/wolfSSL
+../../src/import/tls/wolfssl/IDE/MDK5-ARM/Projects/EchoClient/RTE/wolfSSL
+../../src/import/tls/wolfssl/IDE/IAR-EWARM/embOS/custom_port/wolfcrypt_benchmark_custom_port/Application
+../../src/import/tls/wolfssl/wolfcrypt/src/port/ti
+../../src/import/tls/wolfssl/mcapi/wolfcrypt_test.X/nbproject
+../../src/import/tls/wolfssl/examples/echoserver
+../../src/import/tls/wolfssl/certs/test
+../../src/import/tls/wolfssl/build-aux
+../../examples/mqtt_last_will/bin
+../../src/import/tls/wolfssl/IDE/MDK5-ARM/Projects/EchoServer/RTE/wolfSSL
+../../src/import/tls/wolfssl/mplabx/wolfssl.X/nbproject
+../../src/libxively/mqtt/codec
+../../src/import/tls/wolfssl/wrapper/CSharp/wolfSSL-DTLS-Server/Properties
+../../src/import/tls/wolfssl/cyassl
+../../examples/mqtt_logic_example/bin
+../../src/import/tls/wolfssl/IDE/IAR-EWARM/Projects/test
+../../src/import/tls/wolfssl/IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/Application
+../../src/libxively/memory
+../../src/libxively/io/fs
 ../../src/libxively/debug_extensions/memory_limiter
 ../../src/import/tls/wolfssl/IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_test_SAMV71_XULT/settings
-../../src/import/tls/wolfssl/wolfssl/openssl
+../../src/import/tls/wolfssl/IDE/ARDUINO
+../../src/libxively/io/simplelink
+../../src/import/cmocka/include/cmockery
+../../src/libxively/memory/wmsdk
+../../src/import/tls/wolfssl/IDE/MDK-ARM/LPC43xx
 ../../src/bsp/platform/simplelink
-../../src/bsp/platform/wmsdk
-../../src/bsp/platform/microchip
+../../src/import/tls/wolfssl/IDE/MDK5-ARM/Projects/EchoClient
+../../src/import/tls/wolfssl/mqx
+../../src/import/tls/wolfssl/wolfssl/openssl
+../../src/protofiles
+../../src/import/tls/wolfssl/IDE/iOS/wolfssl_testsuite.xcodeproj
+../../src/import/tls/wolfssl/mqx/wolfcrypt_test/Sources
+../../src/import/tls/wolfssl/tirtos/packages/ti/net/wolfssl/tests/EK_TM4C1294XL/wolfcrypt/test
+../../src/import/tls/wolfssl/wrapper/python
+../../src/import/tls/wolfssl/IDE/LPCXPRESSO
+../../src/libxively/io/fs/dummy
+../../src/import/protobuf-c/library
+../../src/import/tls/wolfssl/examples/server
+../../src/import/tls/wolfssl/mplabx/wolfcrypt_benchmark.X/nbproject
+../../src/import/protobuf-c/test/compiler_generated
 ../../src/bsp/platform/posix
-../../src/bsp/tls/wolfssl
-../../src/bsp/tls/mbedtls
-../../src/experimental/heap_allocator/src
-../../src/models/heap_allocator
-../../src/bsp/platform/cc3200
-../../src/experimental/heap_allocator
+../../src/libxively/mqtt/logic
+../../src/import/tls/wolfssl/IDE/MDK5-ARM/Src
+../../src/tests/itests/tools
+../../src/import/tls/wolfssl/IDE/MDK5-ARM/Projects/wolfSSL-Full/RTE/wolfSSL
```

Comment:
ok this is here because this PR is against the MASTER branch. My mistake I'll try to change, if not possible, then will create another PR.